app/controllers/EPayeStartController.scala

```diff
@@ -87,15 +89,15 @@ class EPayeStartController @Inject() (
         } else {
           data.rejections.head match {
             case YouAlreadyHaveAPaymentPlan => (Messages.NotEligible.`You already have a payment plan with HMRC`, existingPaymentPlanPartial())
```

Comment:
Sorry I did it again with the formatting, this is the main change to allow the file your return page. along with the new twirl template partial